dev/com.ibm.ws.jdbc_fat/test-resource/loginmodule/src/loginmodule/TestGSSCredential.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2018, 2019 IBM Corporation and others.
```

Comment:
Should these be 2018, 2020?  Or does the copyright only change when the code changes?  Or were you just adding in 2018 since that is when the file was created?